src/DaylightSavingTime.cpp

```diff
@@ -67,9 +67,11 @@ void DaylightSavingTime::determineDstStartAndEnd(const tm &givenTm)
     tm dstEndTm = givenTm;
     if (DST_LOCATION == Europe)
     {
+        int offset_minutes = static_cast<int>(UTC_OFFSET * 60);
```

Comment:
```suggestion
        int offsetMinutes = static_cast<int>(UTC_OFFSET * 60);
```
To use the same naming convention as the rest of the project.